PiT 24.6: Project does not work in JDK-21

Clone and run the project in JDK 21

```
git clone https://github.com/vaadin/mpr-demo.git
cd mpr-demo
mvn -ntp -B jetty:run  
```

It fails:

```
Caused by: java.lang.RuntimeException: Error scanning entry com/fasterxml/jackson/core/io/doubleparser/FastDoubleSwar.class from jar file:///home/runner/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.17.2/jackson-core-2.17.2.jar
Caused by: java.lang.IllegalArgumentException: Unsupported class file major version 65
...
at org.eclipse.jetty.annotations.AnnotationParser.scanClass (AnnotationParser.java:896)
```

--- a/pom.xml
+++ b/pom.xml
@@ -18,7 +18,7 @@
         <vaadin8.version>8.24.0</vaadin8.version>
         <flow.version>24.5.5</flow.version>
         <vaadin.plugin.version>${vaadin8.version}</vaadin.plugin.version>
-        <jetty.plugin.version>11.0.24</jetty.plugin.version>
+        <jetty.plugin.version>12.0.15</jetty.plugin.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>17</maven.compiler.source>
         <maven.compiler.target>17</maven.compiler.target>
@@ -205,11 +205,11 @@
             <!-- The Jetty plugin allows us to easily test the development build by
                 running jetty:run on the command line. -->
             <plugin>
-                <groupId>org.eclipse.jetty</groupId>
-                <artifactId>jetty-maven-plugin</artifactId>
+                <groupId>org.eclipse.jetty.ee10</groupId>
+                <artifactId>jetty-ee10-maven-plugin</artifactId>
                 <version>${jetty.plugin.version}</version>
                 <configuration>
-                    <scanIntervalSeconds>2</scanIntervalSeconds>
+                    <scan>2</scan>
                 </configuration>
             </plugin>
         </plugins>

--- a/pom.xml
+++ b/pom.xml
@@ -18,7 +18,7 @@
         <vaadin8.version>8.24.0</vaadin8.version>
         <flow.version>24.5.5</flow.version>
         <vaadin.plugin.version>${vaadin8.version}</vaadin.plugin.version>
-        <jetty.plugin.version>11.0.13</jetty.plugin.version>
+        <jetty.plugin.version>11.0.24</jetty.plugin.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>17</maven.compiler.source>
         <maven.compiler.target>17</maven.compiler.target>

